### PR TITLE
Threescale 1471 links alignment - Right alignment

### PR DESCRIPTION
**Pull request by josemigallas**

### CODE REVIEW

File: app/assets/stylesheets/provider/admin/_dashboard.scss [hunk 1]
```diff
@@ -30,6 +30,7 @@
     color: $font-color;
     min-width: 10%;
     float: left;
+    max-width: 40%;
```
Comment (damianpm) [inline app/assets/stylesheets/provider/admin/_dashboard.scss]: I think 40% is a lot. I'm using `min-width: 10%`;

Comment (josemigallas) [reply app/assets/stylesheets/provider/admin/_dashboard.scss]: However 10% is ineffective:
screen shot 2018-11-05 at 10 03 52

At least I'd go for 25%, it has a better balance:
screen shot 2018-11-05 at 10 04 51

@thomasmaas should have the final word though.

Comment (damianpm) [reply app/assets/stylesheets/provider/admin/_dashboard.scss]: Good point, I haven't tested such a loooong title

Comment (thomasmaas) [reply app/assets/stylesheets/provider/admin/_dashboard.scss]: @damianpm I think we should use ellipsis for long titles here. With full name in title attribute

Comment (damianpm) [reply app/assets/stylesheets/provider/admin/_dashboard.scss]: @thomasmaas @josemigallas 
max-width: 20% and text ellipsis:
ellipsis

**Comment by josemigallas** [comment]
@damianpm
I removed `min-width` because it does not make any difference anymore
I set `max-width` to 25% because the name is too short otherwise, and it does not add any more lines (integrate api is still in the second line only)
I cherry-picked the fix for the unit tests, they should pass now

**Comment by damianpm** [comment]
@josemigallas Should we add Gui or any other ruby guy to review?

**Comment by josemigallas** [comment]
@damianpm I don't think so, it's just a cherry pick from master and also unit tests are passing now.

**Comment by damianpm** [comment]
> @damianpm I don't think so, it's just a cherry pick from master and also unit tests are passing now.

But we still have tests failing

**Pull request merged**